src/pkg/cli/tail.go

```diff
@@ -76,6 +76,7 @@ type TailOptions struct {
 	Verbose            bool
 	LogType            logs.LogType
 	Filter             string
+	WaitTimeout        int
```

Comment:
Doesn't belong here. It's a timeout for `compose up` not for tail.

Reply:
This code begs to differ: https://github.com/DefangLabs/defang/blob/b0ed4d9fc82aa8380ff338a0995250f3055cce4f/src/cmd/cli/command/compose.go#L178-L182

Reply:
You linked to `makeComposeUp`, which is my point?

Reply:
The wait timeout timer cancels the new context we create just for tail